src/pages/AboutYou.js

```diff
@@ -0,0 +1,36 @@
+const { dashboardStyles } = require('../styles.js')
+const { html } = require('../utils.js')
+const Layout = require('../components/Layout.js')
+const LogoutLink = require('../components/LogoutLink.js')
+const SummaryTable = require('../components/SummaryTable.js')
+const ButtonLink = require('../components/ButtonLink.js')
+
+const aboutYouRows = ({ name, address }) => {
+  return [{ key: 'Name', value: name }, { key: 'Mailing address', value: address }]
+}
+
+const AboutYou = ({ data = {} }) =>
+  html`
+    <${Layout}>
+      <div class=${dashboardStyles}>
+        <${LogoutLink} />
+        <h1>About you</h1>
+        <p>
+          This is the current name and address we have on file for you. Please update any
+          out-of-date information and then continue to the next section.
+        </p>
+
+        <${SummaryTable} rows=${aboutYouRows(data)} ifEditable=${true} //>
+        <p>
+          There are <strong>2 sections</strong> remaining, which should take ${' '}<strong
+            >5 minutes</strong
```

Comment:
I'm not sure I fully understand the spacing on this line and the next 2. Is it a pattern I'm not aware of?

Reply:
Oh, that's my editor formatting the line.

Once I put `<strong>` tags around some of the words, it formatted it like:

```
<p>
  There are <strong>2 sections</strong> remaining, which should take
  <strong>5 minutes</strong>
  to complete.
</p>
```

And then on the screen it looked like: "There are **2 sections** remaining, which should take**5 minutes**to complete".

So I added forced whitespace: `${' '}` around the `strong` tags and that's where we ended up with.